### ISSUE TITLE
Capitalization breaks it(?)

### ISSUE THREAD

**Issue by NanoCats** [Puv1s/ColorTooltips#3]
Reproduction steps

1. Be me, experimenting with plugin, write a color in whatever format like so 
![image](https://user-images.githubusercontent.com/14065903/131264053-057cd23a-8862-4905-abfc-f17263e81019.png)

2. Get color [from a site](https://hexcolor.co/hex/55e0bf) which had it in multi formats, wanted to see if it works on two in the same message
3. Entering #55E0BF (Eucalyptus) - RGB 85, 224, 191 causes just the hex code to be accepted, sure, can't account for all formats I guess
4. Try again, discover that RGB being capitalized stops ANY color formatting in this message 
![image](https://user-images.githubusercontent.com/14065903/131264090-906e6171-6725-46fb-adc2-e9f37e089718.png)

I tohught this was enough of a thing that resembles a bug that it was worth reporting.

**Comment by Puv1s** [comment]
Yeah thanks, I'll be modifying the regex so it takes wider range of capitalization and spacings.
